Sources/Edge.swift

```diff
@@ -65,6 +68,15 @@ public class Edge: NSObject, Extension {
             let identitySharedState = getXDMSharedState(extensionName: EdgeConstants.SharedState.Identity.STATE_OWNER_NAME,
                                                         event: event)
 
+            return configurationSharedState?.status == .set && identitySharedState?.status == .set
+        } else if event.isResetIdentitiesEvent {
+            let configurationSharedState = getSharedState(extensionName: EdgeConstants.SharedState.Configuration.STATE_OWNER_NAME,
+                                                          event: event)
+            // use barrier to wait for EdgeIdentity to handle the reset
```

Comment:
so are we still waiting for the reset request and not for the resetComplete?

Reply:
Yeah, we could listen for the event coming from edge identity notifying it has reset or use the barrier flag to wait until we know its done processing